charts/linkerd-control-plane/values.yaml

```diff
@@ -175,6 +175,11 @@ proxy:
 
 # proxy-init configuration
 proxyInit:
+  # -- Variant of iptables that will be used to configure routing. Currently,
+  # proxy-init can be run either in 'nft' or in 'legacy' mode. The mode will
+  # control which utility binary will be called. The host must support
+  # whichever mode will be used
+  iptablesMode: "nft"
```

Comment:
do we expect there to be more modes in the future other than "nft" and "legacy"?  if not, should this be a boolean rather than a string?

Reply:
Initially I chose a boolean. I think it would be reasonable to expect other modes. We might chose to move entirely over to `nftables`, and although it is early to tell, we might decide to support `ebpf` or something similar in the future.